### ISSUE TITLE
Creating new scenarios by copying in GUI doesn't work

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2508]
Steps to reproduce:

- open reference-case-open
- click "+ Create New Scenario"
- enter name (e.g. "second")
- select "Copy input folder from another scenario in the project", use baseline
- click "Create"

Expected: Input editor with a copy of the baseline scenario

Buggy behavior:

![image.png](https://images.zenhubusercontent.com/57b40957e40e5714b16d7331/af8e239e-d722-4381-80d0-c7a2c2448285)

Folder structure of project:

```
C:\Users\darthoma\Documents\CityEnergyAnalyst\projects
λ tree reference-case-open
Folder PATH listing
Volume serial number is 000001C1 C84C:1BEB
C:\USERS\DARTHOMA\DOCUMENTS\CITYENERGYANALYST\PROJECTS\REFERENCE-CASE-OPEN
├───baseline
│   ├───inputs
│   │   ├───building-geometry
│   │   ├───building-metering
│   │   ├───building-properties
│   │   │   └───schedules
│   │   ├───networks
│   │   ├───technology
│   │   │   ├───archetypes
│   │   │   │   └───schedules
│   │   │   ├───lifecycle
│   │   │   ├───systems
│   │   │   └───uncertainty
│   │   ├───topography
│   │   └───weather
│   └───outputs
│       ├───data
│       │   ├───costs
│       │   ├───demand
│       │   ├───emissions
│       │   ├───multicriteria
│       │   ├───occupancy
│       │   ├───optimization
│       │   │   ├───decentralized
│       │   │   ├───master
│       │   │   ├───network
│       │   │   ├───slave
│       │   │   │   ├───gen_0
│       │   │   │   ├───gen_1
│       │   │   │   └───gen_2
│       │   │   └───substations
│       │   ├───potentials
│       │   │   └───solar
│       │   ├───solar-radiation
│       │   └───thermal-network
│       │       └───DH
│       └───plots
│           └───new_basic
│               └───demand
└───second
    └───inputs
        └───networks
```

**Comment by daren-thomas** [comment]
(the networks folder is empty)

**Comment by daren-thomas** [comment]
(this might have been introduced by #2503 - @reyery could you check if this happens in master too?)

**Comment by reyery** [comment]
Fixed in #2503
